etc/tempesta_fw.conf

```diff
@@ -42,50 +76,125 @@
 # Default:
 #   listen 80;
 
-# TAG: cache
+
+# TAG: backend
```

Comment:
We have forward proxy in plans, so let's do not stick to poor reverse proxy in our design and code. So let's move from 'backend' to 'server' hereafter.